iwyu_stl_util.h

```diff
@@ -62,9 +60,8 @@ template <class AssociativeContainer>
 bool ContainsAnyKey(
     const AssociativeContainer& container,
     const set<typename AssociativeContainer::key_type>& keys) {
-  for (typename set<typename AssociativeContainer::key_type>::const_iterator
-           it = keys.begin(); it != keys.end(); ++it) {
-    if (ContainsKey(container, *it))
+  for (const auto &i : keys) {
+    if (ContainsKey(container, i))
```

Comment:
Can we replace `i` with `key`? Or maybe try `std::any_of`? Though I don't know if MSVC 2013 supports `std::any_of`.